docs/source/install/deb.rst

```diff
@@ -19,11 +19,10 @@ APT
 Pip
 '''
 
-The easiest way to install these is to use the requirements.txt file from the |st2| downloads server.  This is kept up to date for the version specified in the path.
+The easiest way to install these is to use the requirements.txt file from the |st2| repository.  This is kept up to date for the version specified in the path.
 
 ::
-
-    curl -q -k -O https://ops.stackstorm.net/releases/st2/0.9dev/requirements.txt
+    curl -q -k -O https://raw.githubusercontent.com/StackStorm/st2/master/requirements.txt
```

Comment:
Right. I believe the missing link is in fact why Kiril brought it up in the first place. 
But, is there a reason why we wouldn't point to the github repo? Is this to try and track downloads?

-James 




On Tue, Mar 17, 2015 at 9:20 AM -0700, "Tomaz Muraus" <notifications@github.com> wrote:












In docs/source/install/deb.rst:
>  
>  ::
> -
> -    curl -q -k -O https://ops.stackstorm.net/releases/st2/0.9dev/requirements.txt
> +    curl -q -k -O https://raw.githubusercontent.com/StackStorm/st2/master/requirements.txt



Your assumption is correct, but requirements should live on ops.stackstorm.net and there should be a 0.9dev directory there.



Edit: I just had a look - looks like we need to update / fix our build workflow to also copy the requirements file to the dist directory - new versions are missing it.



—
Reply to this email directly or view it on GitHub.